pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/readers/forward/FixedByteChunkSVForwardIndexReader.java

```diff
@@ -48,6 +52,179 @@ public ChunkReaderContext createContext() {
     }
   }
 
+  @Override
+  public void fillValues(int[] docIds, int length, int[] values, ChunkReaderContext context) {
```

Comment:
can we name it readValues to maintain symmetry with readDictIds and other read methods

Reply:
Last time I called a method which filled an out-parameter array something like `readValues` I was asked to change it to `fillValues`